setup.py

```diff
@@ -48,7 +48,7 @@ def read(fname):
         'Programming Language :: Python :: 3.5',
     ],
 
-    install_requires=['numpy', 'sagemaker-containers==2.1.0', 'Pillow', 'retrying', 'six'],
+    install_requires=['numpy', 'sagemaker-containers', 'Pillow', 'retrying', 'six'],
```

Comment:
should we put a minimum?

Reply:
+1

Reply:
Talked offline. Set lower bound to 2.1.0